electron/main.ts

```diff
@@ -457,6 +458,14 @@ ipcMain.handle('moveInstall', async (event, [appName, path]: string[]) => {
     .catch(console.log)
 })
 
+ipcMain.handle('changeInstall', async (event, [appName, newPath]: string[]) => {
```

Comment:
I think this should be called `changeInstalPath` since `changeInstall` could be ambiguous. :)